docs-ref-conceptual/interactive-azure-cli.md

```diff
@@ -14,111 +14,162 @@ Keywords: azure cli interactive mode
 
 # Azure CLI interactive mode
 
-You can use Azure CLI in interactive mode by running the `az interactive` command.  The Azure CLI interactive mode places you in an interactive shell with autocompletion, command descriptions, and examples.
+## What is the Azure CLI interactive mode?
 
-![interactive mode](./media/interactive-azure-cli/webapp-create.png)
+Azure CLI Interactive Mode (az interactive) provides users an interactive environment to run Azure CLI commands. The interactive mode makes it easier for you to learn the Azure CLI’s capabilities, command syntax, and output formats. It provides autocompletion dropdowns, auto-cached suggestions combined with runtime documentation, and includes examples about how each command is used. Azure CLI Interactive Mode aims to provide an ideal experience for users learning to use Azure CLI commands. 
 
-> [!NOTE]
-> We're not using the default style here, which doesn't read as well on a black background.
+## Prerequisites
 
-If you're not already signed in to your account, use the `login` command.
+Use the Bash or PowerShell environment in [Azure Cloud Shell](/azure/cloud-shell/overview). For more information, see [Quickstart for Bash in Azure Cloud Shell](/azure/cloud-shell/quickstart).
```

Comment:
We talked about this, and I think we're on the right track, but it is odd to then show all PowerShell 7 screen prints. Do you agree?  Would this work: "Use a Bash or PowerShell environment in [Azure Cloud Shell]() or [run the Azure CLI locally](./install-azure-cli-windows&tabs=powershell) in PowerShell 7.4.1 or higher"  (whatever version of PowerShell you tested in).